storage-resize-images/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Version 0.1.18
+
+fixed - capitalised file extensions (#549)
+
+fixed - Removed code coverage check on ci
```

Comment:
please remove this (this is a repo maintenance detail, not related to the functionality of the extension)